packages/inter-protocol/test/auction/test-auctionContract.js

```diff
@@ -906,6 +924,8 @@ test.serial('onDeadline exit', async t => {
     currentPriceLevel: { numerator: { value: 9_350_000_000_000n } },
   });
 
+  await snapshotStorage(t, driver.mockChainStorage);
```

Comment:
each test is defining its own `snapshotStorage` but they follow the same pattern. please DRY it out. I think something like,
```suggestion
  await snapshotStorage(t, driver.mockChainStorage, {'mockChainStorageRoot.thisAuction': 'published.auction'}, `Under "published", the "auction" node is delegated to
the auctioneer contract, which publishes data as follows.
See also board marshalling conventions (_to appear_).
` );
```

or maybe more to the point: `documentStorageKeys` since the `note` and the rewriting to `published:` is not really about testing.